docs/metrics/getting-started-prometheus-grafana/getting-started-prometheus-grafana.csproj

```
@@ -1,5 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
-  <ItemGroup>
-    <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Exporter.Prometheus\OpenTelemetry.Exporter.Prometheus.csproj" />
-  </ItemGroup>
+<PropertyGroup>
+  <TargetFrameworks>netstandard2.0</TargetFrameworks>
+</PropertyGroup>
+<ItemGroup>
+  <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Exporter.Prometheus.HttpListener\OpenTelemetry.Exporter.Prometheus.HttpListener.csproj" />
+</ItemGroup>
```

I think some indentation was lost

```suggestion
  <PropertyGroup>
    <TargetFrameworks>netstandard2.0</TargetFrameworks>
  </PropertyGroup>
  <ItemGroup>
    <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Exporter.Prometheus.HttpListener\OpenTelemetry.Exporter.Prometheus.HttpListener.csproj" />
  </ItemGroup>
```